nrf_wifi/fw_if/umac_if/src/fmac_util.c

```diff
@@ -363,7 +363,8 @@ enum nrf_wifi_status nrf_wifi_check_mode_validity(unsigned char mode)
 bool nrf_wifi_util_is_rawpktmode_enabled(struct nrf_wifi_fmac_vif_ctx *vif)
 {
 	if ((vif->if_type == NRF_WIFI_STA_TX_INJECTOR) ||
-	    (vif->if_type == NRF_WIFI_MONITOR_TX_INJECTOR)) {
+	    (vif->if_type == NRF_WIFI_MONITOR_TX_INJECTOR) ||
+	    (vif->if_type == NRF_WIFI_STA_PROMISC_TX_INJECTOR)) {
```

Comment:
Shouldn't these be protected by #if defined(CONFIG_NRF700X_RAW_DATA_RX) || defined(CONFIG_NRF700X_PROMISC_DATA_RX)

Reply:
No ...this only checks for TX injection and only comes in when config block TX injection is enabled, be it with Promiscuous or Monitor. So this is fine. No check is required